patches/p1.js

```diff
@@ -0,0 +1,36 @@
+const Enmap = require("enmap");
+
+const v1Roles = new Enmap({name :'rolesData'});
+const v2Roles = new Enmap({name :'rolesDataV2'});
+/**
+ * Migration for the roles command's schema
+ */
+function migrateRolesToV2() {
+  console.log(`rolesV1-pre: ${v1Roles.export()}`);
+  console.log(`rolesV2-pre: ${v2Roles.export()}`);
+
+  //v1 formatted as guildId -> arr[roleNames]
+  //v2 is a documentStore of arbitraryId -> {roleInfo} e.g. name, guildId, etc. 
+  v1Roles.keyArray().forEach(guildId => {
```

Comment:
also just to make sure you're aware - v1 doesn't have any of the #welcome channel data thats all hardcoded

Reply:
yup!